src/js/BookReader/Mode2Up.js

```diff
@@ -99,17 +99,21 @@ export class Mode2Up {
   /**
    * Checks to see if the images/pages in view
    * are of equal or better quality
+   * If not, we return yes
    *
    * @returns {Boolean}
    */
   get shouldRedrawSpread() {
-    const { prefetchedImgs, twoPage } = this.br;
+    const { prefetchedImgs, displayedIndices } = this.br;
     const { reduce: idealReductionFactor } = this.getIdealSpreadSize( this.br.twoPage.currentIndexL, this.br.twoPage.currentIndexR );
 
     const currentImagesAreLarger = this.br.reduce <= idealReductionFactor;
-    const pagesInViewArePrefetched = prefetchedImgs[twoPage.currentIndexL] && prefetchedImgs[twoPage.currentIndexR];
+    const leftDisplayed = prefetchedImgs[displayedIndices[0]] || {};
+    const rightDisplayed = prefetchedImgs[displayedIndices[1]] || {};
+    const LeftImgIsPrefetchedToScale = leftDisplayed && (leftDisplayed.reduce <= this.br.reduce);
+    const RightImgIsPrefetchedToScale = rightDisplayed && (rightDisplayed.reduce <= this.br.reduce);
```

Comment:
updating this to check currently displayed pages - if images are of decent size given a view, then let's keep them